SimPEG/utils/pgi_utils.py

```diff
@@ -115,13 +123,14 @@ def make_SimplePGI_regularization(
         gmm=gmm,
```

Comment:
It's a bit of a choice here, but we could just ask for kwargs and refer to the inputs of the PGI regularization, instead of duplicating all the input parameters. @jcapriot what's your opinion on this?